capstone/src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -38,20 +38,27 @@
   private static final String KEY = "API_KEY";
   private static final String NUM_RESULTS = "5";
   private static final MediaType JSON = MediaType.get("application/json; charset=utf-8");
+  private String postRequestBodyData;
+  private String channelId;
+  private String videoId; 
```

Comment:
Once the methods are combined, the first variable here can be removed. Also are the second two variables used?

Reply:
The variables are used depending on the search type the user selects. This Sevrlet requires extra variables because it has the ability to make requests to multiple URL's.

Reply:
Although the servlet is making requests to possibly different URLs, it looks like `channelId` and `videoId` aren't used in this servlet's function. I would recommend that we can delete them.